float64 spring_stiffness
float64 damping_coefficient
---
bool success
string message
